docs/conf.py

```diff
@@ -142,3 +142,7 @@
                     'https://www.sdss.org']
 linkcheck_timeout = 180
 linkcheck_anchors = False
+
+# Enable nitpicky mode - which ensures that all references in the docs resolve.
+nitpicky = True
+nitpick_ignore = [('py:obj', 'stsynphot')]
```

Comment:
I don't think you have to ignore this anymore since you use double backticks now?
```suggestion
nitpick_ignore = []
```